### DISCUSSION TITLE
Missed attachments when export the organizations

### DISCUSSION THREAD

**Discussion by tainv-93** [dani-garcia/vaultwarden#3338] [Q&A]
### Deployment environment

* vaultwarden version:

* Install method:

* Clients used: 

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

**Comment by BlackDex** [comment]
Attachments are not part of the export, not for Organizations, not for Users.
Also, this is a Client Side feature, not something Vaultwarden can fix.

I think there are multiple features requests for this already at Bitwarden.
There is a tool called [Portwarden](https://github.com/vwxyzjn/portwarden/) which could help.
